types/rangy/tsconfig.json

```diff
@@ -19,7 +19,8 @@
     },
     "files": [
         "index.d.ts",
-        "rangy-classapplier.d.ts",
+        "lib/rangy-classapplier.d.ts",
```

Comment:
Also enable `strictNullChecks` since this is a major change to the library

Reply:
done